File: LeetCode/C#/1502. Can Make Arithmetic Progression From Sequence.cs
public class Solution {
    public bool CanMakeArithmeticProgression(int[] arr) {
        Array.Sort(arr);
        var diff = arr[1] - arr[0];
        
        for(var i = 1; i < arr.Length - 1; i++)
        {
            if(arr[i + 1] - arr[i] != diff)
            {
                return false;
            }
        }
        
        return true;
    }
}

File: LeetCode/Programming Skills - Java/1502. Can Make Arithmetic Progression From Sequence.java
class Solution {
    public boolean canMakeArithmeticProgression(int[] arr) {
        if (arr.length == 2) {
            return true;
        } else {
            Arrays.sort(arr);

            int difference = arr[1] - arr[0];

            for (int i = 2; i < arr.length; i++) {
                if (arr[i] - arr[i - 1] != difference) {
                    return false;
                }
            }

            return true;
        }
    }
}